custom-metrics-stackdriver-adapter/README.md

```diff
@@ -45,14 +45,19 @@ them to scale your application, following [HPA walkthrough].
      - To set scopes in existing clusters you can use `gcloud beta compute
        instances set-scopes` command, see [gcloud
        documentation](https://cloud.google.com/sdk/gcloud/reference/beta/compute/instances/set-scopes).
+    * On GKE, you need cluster-admin permissions on your cluster. You can grant
+      your user account this permissions with following command:
+      ```
+      kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user $(gcloud config get-value account)
+      ```
 
 1. Start *Custom Metrics - Stackdriver Adapter*.
 
 ```sh
-kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/adapter-beta.yaml
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/deploy/production/adapter.yaml
 ```
 
-### Export metrics to Stackdriver
+### Export custom metrics to Stackdriver
```

Comment:
I'm wary of using "custom" too much. It is a bit confusing as there are custom metrics in Kubernetes, in Stackdriver and there are external metrics which are also custom in some sense. I think it's not so important that they are custom. WDYT?